src/frontend/components/StudentLiveAdvertising/StudentLiveDescription/index.tsx

```diff
@@ -22,16 +33,18 @@ export const StudentLiveDescription = ({
         textAlign="center"
         size="small"
```

Comment:
```suggestion
        textAlign="center"
        size="small"
        style={{ fontStyle: video.title ? undefined : 'italic' }}
```
IMO, it's a bit confusing to have the same policy / font style for the informational header 'Live is starting' and the message 'The title has not started yet.'. Maybe applying an italic style could help gain clarity ? WDYT ?

![image](https://user-images.githubusercontent.com/83247226/165971641-40002266-c39c-4896-a9bf-75fa68bceb7f.png)